scipy/sparse/linalg/eigen/_svds.py

```diff
@@ -221,12 +225,39 @@ def matmat_XH_X(x):
         eigvals, eigvec = lobpcg(XH_X, X, tol=tol ** 2, maxiter=maxiter,
                                  largest=largest)
 
+    elif solver == 'propack':
+        from scipy.sparse.linalg import svdp
```

Comment:
I see why this was done, but I'd suggest at the top:
```python3
import scipy.sparse.linalg as spla
```
and below call `spla.svdp`.

Reply:
Done